tools/tpm2_create.c

```diff
@@ -220,9 +266,10 @@ bool tpm2_tool_onstart(tpm2_options **opts) {
       { "pubfile",              required_argument, NULL, 'u' },
       { "privfile",             required_argument, NULL, 'r' },
       { "context-parent",       required_argument, NULL, 'C' },
+      { "context",              required_argument, NULL, 'c' },
```

Comment:
`-c`/`--context` is by convention a child context for a command, would this be better as `-o`/`--out-context` like `tpm2_load` ?

Reply:
Yeah I thought about this as well, initially  I wasn't happy with -o option on tpm2_load but considering you can load a child or parent, it makes more sense.

Reply:
Actually, -c is really "this object context". For instance tpm2_readpublic, where as -C is this objects parent.

Reply:
Ill keep it same as tpm2_load and we can figure out the proper ones and be consistent when we go through all the options for release.